src/main/java/org/hisp/dhis/model/Style.java

```diff
@@ -0,0 +1,9 @@
+package org.hisp.dhis.model;
+
+public enum Style
```

Comment:
You think we could rename this class to `FormStyle` since it is in the general model package? This way it becomes a bit more specific and we avoid collisions if there are other style classes.